ci-operator/config/openshift/origin/openshift-origin-main.yaml

```diff
@@ -318,16 +318,53 @@ tests:
       - observers-resource-watch
     workflow: openshift-e2e-aws-serial
 - always_run: false
-  as: e2e-aws-ovn-virt-techpreview
+  as: e2e-metal-ipi-ovn-virt-dualstack
```

Comment:
hmm so we are removing aws-virt and replacing it all with metal-virt?

Reply:
Any reason to not keep this one as AWS? It seems to not required dualstack, localnet or bgp?